source/core-concepts/notebooks.rst

```diff
@@ -0,0 +1,11 @@
+.. meta::
+    :description: What are Valohai hosted notebooks? Explore your data and model in ephemeral environments.
+
+Notebooks
+=========
+
+**A notebook** is a hosted, short-lived Jupyter environment for experimenting with your model.
+
+Notebooks are hosted by Valohai, with the Jupyhai extension installed to allow you to seamlessly run your notebook contents as a Valohai execution.
+
+More documentation forthcoming.
```

Comment:
I would leave this out, as it doesn't add any value

Reply:
fixed